src/pages/waiting-room/waiting-room.analytics.effects.ts

```diff
@@ -109,4 +113,23 @@ export class WaitingRoomAnalyticsEffects {
     }),
   );
 
+  @Effect()
+  waitingRoomCBTSubmitted$ = this.actions$.pipe(
+    ofType(WAITING_ROOM_CBT_SUBMITTED),
```

Comment:
Could we not run this from the CbtNumberChanged event ?

Reply:
src/modules/tests/pre-test-declarations/cat-a/pre-test-declarations.cat-a.actions.ts

Reply:
Assuming the reason its here is so we only capture confirmed category selections?